includes/blueprints/fetch.php

```diff
@@ -10,7 +10,46 @@
 use WP_Error;
 
 /**
- * Downloads a blueprint zip file from the specified URL.
+ * Gets blueprint from either local or remote path.
+ *
+ * @param string $path The path to the file.
+ * @return string
+ */
+function get_blueprint( string $path ) {
+	if ( filter_var( $path, FILTER_VALIDATE_URL ) ) {
+		return get_remote_blueprint( $path );
+	} else {
+		return get_local_blueprint( $path );
+	}
+}
+
+/**
+ * Downloads a blueprint zip file from the specified local path.
+ *
+ * @param string $path The path to the zip file.
+ * @return string|WP_Error
+ */
+function get_local_blueprint( string $path ) {
+	if ( ! file_exists( $path ) ) {
+		return new WP_Error(
+			'acm_blueprint_invalid_file_path',
+			esc_html__( 'File path was invalid.', 'atlas-content-modeler' ),
+			[ 'status' => 400 ]
+		);
+	}
+
+	if ( ! is_readable( $path ) ) {
+		return new WP_Error(
+			'acm_blueprint_file_not_readable',
+			esc_html__( 'Received empty response body.', 'atlas-content-modeler' )
```

Comment:
This was leftover from the copy paste I think. Probably should read something like this. Anyone have a better suggestion for error text?

```suggestion
			esc_html__( 'File not found or not readable.', 'atlas-content-modeler' )
```